content/kubermatic/main/installation/upgrading/upgrade-from-2.21-to-2.22/_index.en.md

```diff
@@ -0,0 +1,227 @@
+
+{{% notice note %}}
+This step can be skipped if User Cluster MLA was not installed previously.
+{{% /notice %}}
+
+Between KKP 2.21 and 2.22, the installation method for [User Cluster MLA]({{< ref "../../../architecture/monitoring-logging-alerting/user-cluster/" >}}) has changed and is now part of `kubermatic-installer`. Updated installation instructions can be found [here]({{< ref "../../../tutorials-howtos/monitoring-logging-alerting/user-cluster/admin-guide/" >}}).
+
+User Cluster MLA should be upgraded after KKP has been upgraded to 2.22. This has to be done for each Seed that has User Cluster MLA installed.
+
+#### Helm Values File
+
+Providing a Helm values file for the User Cluster MLA installation is optional and depends on whether you have passed any non-standard configuration values to your MLA setup in earlier versions (e.g. to set the `StorageClass` for some components) or have set up IAP, you will need to merge all custom Helm values into a shared `mlavalues.yaml` file, similar to the `values.yaml` provided to `kubermatic-installer` for installing a KKP setup. For example, a file configuring IAP and custom Cortex storage would looks like this:
+
+```yaml
+# This is a example file, do not use it!
+#
+# Cortex configuration
+cortex:
+  compactor:
+    persistentVolume:
+      storageClass: kubermatic-slow
+  store_gateway:
+    persistentVolume:
+      storageClass: kubermatic-slow
+  ingester:
+    persistentVolume:
+      storageClass: kubermatic-slow
+  alertmanager:
+    persistentVolume:
+      storageClass: kubermatic-slow
+
+# IAP configuration
+# this section is only needed if IAP was configured for MLA before
+iap:
+  oidc_issuer_url: <OIDC Issuer URL>
+  deployments:
+    grafana:
+      <Grafana IAP configuration>
+    alertmanager:
+      <Alertmanager IAP configuration>
+```
+
+#### Running the Upgrade
+
+{{% notice warning %}}
+Upgrading User Cluster MLA is **briefly disruptive to Consul and Cortex availability**. Consider this when planning the
+upgrade.
+{{% /notice %}}
+
+If a custom values file is required and is ready for use, `kubermatic-installer` can be used to upgrade User Cluster MLA. Ensure that you
+uncomment the command flags that you need (e.g. `--helm-values` if you have a `mlavalues.yaml` to pass and `--mla-include-iap` if you are
+using IAP for MLA; both flags are optional).
+
+```sh
+./kubermatic-installer deploy usercluster-mla \
+  # uncomment if you are providing non-standard values
+  # --helm-values mlavalues.yaml \
+  # uncomment if you deployed MLA IAP before as well
+  # --mla-include-iap \
+  --config kubermatic.yaml
+```
+
+## Post-Upgrade Considerations
+
+
+### KubeVirt Migration
+
+KubeVirt graduates to GA in KKP 2.22 and has gained several new features. However, KubeVirt clusters need to be migrated after the KKP 2.22 upgrade. [Instructions are available in KubeVirt provider documentation]({{< ref "../../../architecture/supported-providers/kubevirt/kubevirt#migration-from-kkp-221" >}}).
```

Comment:
I would change this a little bit that people are not confused about KubeVirt itself.

```suggestion
Kubermatic Virtualization graduates to GA in KKP 2.22 and has gained several new features. However, KubeVirt clusters need to be migrated after the KKP 2.22 upgrade. [Instructions are available in KubeVirt provider documentation]({{< ref "../../../architecture/supported-providers/kubevirt/kubevirt#migration-from-kkp-221" >}}).
```

Reply:
Is that a name we've been using for KubeVirt support anywhere else? I can change it to `KubeVirt support graduates ...` but I myself wouldn't be sure what Kubermatic Virtualization is.

Reply:
Yes, it's been officialy used for KubeVirt.
https://www.kubermatic.com/services/accelerators/virtualization-by-kubevirt/

Reply:
but let me ask @csengerszabo..

@csengerszabo do we stick to this name for KubeVirt still? 
On some pages we have Kubermatic Virtualization on other pages we just refer to KubeVirt, so any opinion on that?

Reply:
Changed it to

> KubeVirt cloud provider support graduates to GA in KKP ...

Let me know if that works for you!

Reply:
Just write: KubeVirt (a.k.a Kubermatic Virtualization)

Documentation can handle a lot :)